creator-node/src/monitors/redis.js

```diff
@@ -0,0 +1,44 @@
+const redis = require('../redis')
+
+/**
+ * Parses the result from a redis `info` command
+ * @param {string} info
+ */
+const parseInfo = (info) => {
```

Comment:
interesting...... there's no built in util for this?

Reply:
drove me insane, but it doesn't look like the redis lib (redisio) or any of the other ones expose this. redisio does it on startup, but doesn't expose...

Reply:
omg lol can't believe we have to do this by hand 🤦‍♂️